mypy/stubtest.py

```diff
@@ -205,10 +205,15 @@ def test_module(module_name: str) -> Iterator[Error]:
     """
     stub = get_stub(module_name)
     if stub is None:
-        runtime_desc = repr(sys.modules[module_name]) if module_name in sys.modules else "N/A"
-        yield Error(
-            [module_name], "failed to find stubs", MISSING, None, runtime_desc=runtime_desc
-        )
+        if "." in module_name:
+            last_part_of_module_name = module_name.split(".")[-1]
+        else:
+            last_part_of_module_name = module_name
+        if not is_probably_private(last_part_of_module_name):
```

Comment:
I might be missing something obvious, but can this just be:
```suggestion
        if not is_probably_private(module_name.split(".")[-1]):
```

Reply:
LOL, yep, that works! Somehow I forgot how `str.split()` works :)